### PR TITLE
blog: add July 2022 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**

### CODE REVIEW

File: docs/blog/newsletter-july-2022.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+And another thing! Referring to other documents
+-----------------------------------------------
+
+After a workshop on accessibility at Write the Docs Portland, there was a discussion on one particular point of language: how to point the reader to a different heading or topic.
+
+The first option, "see", has plenty of advocates. It's a short, simple word and we generally prefer simple language. It's pretty general and doesn't prescribe how you should be using the linked resource (unlike, for example, "read"). "See" is favored by the 2011 IBM Style Guide and the `Google dev docs style guide <https://developers.google.com/style/word-list?hl=en#letter-s>`__.
+
+But "see" has some downsides: it doesn't apply to folks with visual impairments and its vagueness can make it hard to translate correctly.
+
+The next contender is "refer". Per the `ASD-STE100 Simplified Technical English specification <https://asd-ste100.org/request.html>`_, refer is more correct: it means "to tell a person where to find information/give information," while see is litera_lly "to know with the eyes."
```
Comment (hillaryfraley) [inline docs/blog/newsletter-july-2022.rst]: Does the link 
```
`ASD-STE100 Simplified Technical English specification <https://asd-ste100.org/request.html>`_ 
```
need a second underscore at the end?

File: docs/blog/newsletter-july-2022.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Toward the end of the wide-ranging conversation, folks dug into the question of documentation's role in persuading users (in addition to informing and instructing). While some pointed out that this role is conventionally associated with marketing and that marketing language tends to get in the way of the primary goals of informing and instructing, the consensus was that good documentation can and does persuade users to either start with a product or continue with it. Documentation does all the things! ... which might be part of why it's so difficult to provide it with a useful theoretical foundation.
+
+------------------
+What we’re reading
+------------------
+
+The #bipoc group’s been discussing the following materials on diversity, inclusion, and equity. Want to join the conversation? Please join us in the `#bipoc Slack channel <https://writethedocs.slack.com/archives/C016STMEWJD>`_!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-july-2022.rst]: Hm these links are formatted with a single underscore -- maybe the double underscores above are a linting thing?

File: docs/blog/newsletter-july-2022.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Write the Docs Newsletter – July 2022
+#########################################
+
+Hi everyone. Aaron and the newsletter team checking in and hoping all is well for you.
+
+Job security and transitions was a topic on lots of people's minds this month. Some people have been discussing how the Slack community has been a big help to them. Several members have been using it for networking, which helped them find new jobs after layoffs.
+
+In related news, the `2021 Salary Survey <https://www.writethedocs.org/surveys/salary-survey/2021/>`__ has been released. It's based on responses from 959 documentarians (887 employees and 72 independent contractors) from 49 different countries. This year for the first time, we were able to break down median salary by years of experience, gender identity, and organization size. We hope you can find something useful for you in your own salary discussions. And look out for the 2022 survey, which will be opened up for responses later this year.
+
+We hope the post-solstice time, whether summer or winter, is treating you well and we'll be back with more news following a one-month break.
```
Comment (bethaitman) [inline docs/blog/newsletter-july-2022.rst]:
```suggestion
We hope the post-solstice time, whether summer or winter, is treating you well and we'll be back with more news in September following a one-month break.
```

File: docs/blog/newsletter-july-2022.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+The #bipoc group’s been discussing the following materials on diversity, inclusion, and equity. Want to join the conversation? Please join us in the `#bipoc Slack channel <https://writethedocs.slack.com/archives/C016STMEWJD>`_!
+
+A short read: Google, and other big tech companies, are struggling with `caste discrimination <https://www.nbcnews.com/news/asian-america/big-techs-big-problem-also-best-kept-secret-caste-discrimination-rcna33692>`_ after cancelling a Dalit History Month talk.
+
+A medium read: Belonging to a community of support can help shape your career and your life. However, for Black people, it can be hard to find role models in the tech field. The New Stack writes about `BlackTechTwitter <https://thenewstack.io/blacktechtwitter-for-life/>`_ and includes a list of accounts to follow.
+
+A longer read: Textio released their `Language Bias in Performance Feedback report <https://www.oprahdaily.com/life/a32893726/what-is-juneteenth/>`_. They found that when looking at job performance feedback, patterns of bias appear for most all underrepresented groups.
+
+----------------
+From our sponsor
```
Comment (bethaitman) [inline docs/blog/newsletter-july-2022.rst:86]: Just to confirm this will get filled in before shipping

**Review by bethaitman — APPROVED** [review]

File: docs/blog/newsletter-july-2022.rst (hunk 1 above)
Comment (CollierCZ) [reply docs/blog/newsletter-july-2022.rst]: Not sure if it's necessary, but better to keep it consistent, for sure.

File: docs/blog/newsletter-july-2022.rst (hunk 4 above)
Comment (CollierCZ) [reply docs/blog/newsletter-july-2022.rst:86]: Yep. Just did :slightly_smiling_face:

**Pull request merged**